src/components/ErrorHandling/Bugsnag.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Bugsnag, { Event as BugsnagEvent } from "@bugsnag/js";
+import BugsnagPluginReact from "@bugsnag/plugin-react";
+import {
+  getAppVersion,
+  getBugsnagApiKey,
+  getReleaseStage,
+} from "utils/environmentVariables";
+import { DefaultErrorBoundary } from "./ErrorBoundary";
+import ErrorFallback from "./ErrorFallback";
+
+const initializeBugsnag = () => {
+  try {
+    Bugsnag.start({
+      apiKey: getBugsnagApiKey(),
+      appVersion: getAppVersion(),
+      plugins: [new BugsnagPluginReact()],
+      releaseStage: getReleaseStage(),
+    });
+  } catch (e) {
+    console.error("Failed to initialize Bugsnag", e);
+  }
+};
+
+const fallback = () => <ErrorFallback />;
+
+const ErrorBoundary: React.FC<{ children: React.ReactNode }> = ({
+  children,
+}) => {
+  const bugsnagPlugin = Bugsnag.getPlugin("react");
+
+  const Boundary =
```

Comment:
I'm a little hesitant to do this because Bugsnag has the requirement for this function to be called after `Bugsnag.start()`, and I feel like having it here better guarantees that:

<img width="404" alt="Screenshot 2023-09-26 at 10 19 03 AM" src="https://github.com/evergreen-ci/parsley/assets/47064971/02bd7a65-6dbe-44e5-9747-aafccec418b6">

Also this code will be entirely deleted in November